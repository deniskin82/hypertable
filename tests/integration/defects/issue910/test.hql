USE "/";

# delete a cell which does not exist (the delete timestamp is older than
# all other cells
DELETE Field1 FROM LoadTest WHERE ROW = "0000001" TIMESTAMP "2012-08-16 09:22:50.455921030";

SELECT * FROM LoadTest LIMIT 2;
SELECT * FROM LoadTest LIMIT 2 RETURN_DELETES;

SELECT * FROM LoadTest LIMIT 5;
SELECT * FROM LoadTest LIMIT 5 RETURN_DELETES;

SELECT * FROM LoadTest LIMIT 10;
SELECT * FROM LoadTest LIMIT 10 RETURN_DELETES;

SELECT * FROM LoadTest OFFSET 1 LIMIT 5;
SELECT * FROM LoadTest OFFSET 1 LIMIT 5 RETURN_DELETES;
