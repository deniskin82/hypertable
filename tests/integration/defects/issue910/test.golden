
Welcome to the hypertable command interpreter.
For information about Hypertable, visit http://hypertable.com

Type 'help' for a list of commands, or 'help shell' for a
list of shell meta commands.

USE "/";

# delete a cell which does not exist (the delete timestamp is older than
# all other cells
DELETE Field1 FROM LoadTest WHERE ROW = "0000001" TIMESTAMP "2012-08-16 09:22:50.455921030";

SELECT * FROM LoadTest LIMIT 2;
0000001	Field1	8FRg/KO2CTbAH24glsco
0000001	Field2	J/L0+DurZzdKAj0AZmS1
0000001	Field3	cKYDbYvGNdqz307A4pHy
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy
SELECT * FROM LoadTest LIMIT 2 RETURN_DELETES;
0000001	Field1		DELETE COLUMN FAMILY
0000001	Field1	8FRg/KO2CTbAH24glsco
0000001	Field2	J/L0+DurZzdKAj0AZmS1
0000001	Field3	cKYDbYvGNdqz307A4pHy
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy

SELECT * FROM LoadTest LIMIT 5;
0000001	Field1	8FRg/KO2CTbAH24glsco
0000001	Field2	J/L0+DurZzdKAj0AZmS1
0000001	Field3	cKYDbYvGNdqz307A4pHy
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy
0000003	Field1	/J0t0jg1YEBJDQHbCixW
0000003	Field2	OPvQm03zjv9Lq2fgPkqn
0000003	Field3	Y3hLyea2gMgMpLsM4B0L
0000004	Field1	qUCWcbYquc3O/KXUJ0m3
0000004	Field2	inu5Qkfo/RPEKQNs0vDz
0000004	Field3	paKjwIVOpZWI3dNOkoH4
0000005	Field1	OZzPWB6dULQ/9xDNxOZp
0000005	Field2	gPkqnRHGP3AgS0P2I7Yt
0000005	Field3	VNPLdZmad8qVY3hLyea2
SELECT * FROM LoadTest LIMIT 5 RETURN_DELETES;
0000001	Field1		DELETE COLUMN FAMILY
0000001	Field1	8FRg/KO2CTbAH24glsco
0000001	Field2	J/L0+DurZzdKAj0AZmS1
0000001	Field3	cKYDbYvGNdqz307A4pHy
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy
0000003	Field1	/J0t0jg1YEBJDQHbCixW
0000003	Field2	OPvQm03zjv9Lq2fgPkqn
0000003	Field3	Y3hLyea2gMgMpLsM4B0L
0000004	Field1	qUCWcbYquc3O/KXUJ0m3
0000004	Field2	inu5Qkfo/RPEKQNs0vDz
0000004	Field3	paKjwIVOpZWI3dNOkoH4
0000005	Field1	OZzPWB6dULQ/9xDNxOZp
0000005	Field2	gPkqnRHGP3AgS0P2I7Yt
0000005	Field3	VNPLdZmad8qVY3hLyea2

SELECT * FROM LoadTest LIMIT 10;
0000001	Field1	8FRg/KO2CTbAH24glsco
0000001	Field2	J/L0+DurZzdKAj0AZmS1
0000001	Field3	cKYDbYvGNdqz307A4pHy
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy
0000003	Field1	/J0t0jg1YEBJDQHbCixW
0000003	Field2	OPvQm03zjv9Lq2fgPkqn
0000003	Field3	Y3hLyea2gMgMpLsM4B0L
0000004	Field1	qUCWcbYquc3O/KXUJ0m3
0000004	Field2	inu5Qkfo/RPEKQNs0vDz
0000004	Field3	paKjwIVOpZWI3dNOkoH4
0000005	Field1	OZzPWB6dULQ/9xDNxOZp
0000005	Field2	gPkqnRHGP3AgS0P2I7Yt
0000005	Field3	VNPLdZmad8qVY3hLyea2
0000006	Field1	AQpbgPdxPPEE0fhCCHXF
0000006	Field2	lta9Jrd1qSo1ypkCzfbN
0000006	Field3	bYvGNdqz307A4pHykJ69
0000007	Field1	tq1MlIi/tW/CCG6k3MnJ
0000007	Field2	zJiNBW6aCcMKk/SyuReM
0000007	Field3	NhPdYa3Pvwec4yQMUCa8
0000008	Field1	+GF73Xj4aa7yqpV5zfGV
0000008	Field2	L92C5q4mZinu5Qkfo/RP
0000008	Field3	2Luul3HqmYbGhNA6HBUl
0000009	Field1	ykotq1MlIi/tW/CCG6k3
0000009	Field2	M1K49+VEluovX1RtlcuJ
0000009	Field3	JrJgjYC44YbjZkrqbU6e
0000010	Field1	Lu1dHUxsXPAvbaTPbQ2w
0000010	Field2	vKlm8052IrFnZNBQ5Gq1
0000010	Field3	qe1BUZFnAjM01eBJXagO
SELECT * FROM LoadTest LIMIT 10 RETURN_DELETES;
0000001	Field1		DELETE COLUMN FAMILY
0000001	Field1	8FRg/KO2CTbAH24glsco
0000001	Field2	J/L0+DurZzdKAj0AZmS1
0000001	Field3	cKYDbYvGNdqz307A4pHy
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy
0000003	Field1	/J0t0jg1YEBJDQHbCixW
0000003	Field2	OPvQm03zjv9Lq2fgPkqn
0000003	Field3	Y3hLyea2gMgMpLsM4B0L
0000004	Field1	qUCWcbYquc3O/KXUJ0m3
0000004	Field2	inu5Qkfo/RPEKQNs0vDz
0000004	Field3	paKjwIVOpZWI3dNOkoH4
0000005	Field1	OZzPWB6dULQ/9xDNxOZp
0000005	Field2	gPkqnRHGP3AgS0P2I7Yt
0000005	Field3	VNPLdZmad8qVY3hLyea2
0000006	Field1	AQpbgPdxPPEE0fhCCHXF
0000006	Field2	lta9Jrd1qSo1ypkCzfbN
0000006	Field3	bYvGNdqz307A4pHykJ69
0000007	Field1	tq1MlIi/tW/CCG6k3MnJ
0000007	Field2	zJiNBW6aCcMKk/SyuReM
0000007	Field3	NhPdYa3Pvwec4yQMUCa8
0000008	Field1	+GF73Xj4aa7yqpV5zfGV
0000008	Field2	L92C5q4mZinu5Qkfo/RP
0000008	Field3	2Luul3HqmYbGhNA6HBUl
0000009	Field1	ykotq1MlIi/tW/CCG6k3
0000009	Field2	M1K49+VEluovX1RtlcuJ
0000009	Field3	JrJgjYC44YbjZkrqbU6e
0000010	Field1	Lu1dHUxsXPAvbaTPbQ2w
0000010	Field2	vKlm8052IrFnZNBQ5Gq1
0000010	Field3	qe1BUZFnAjM01eBJXagO

SELECT * FROM LoadTest OFFSET 1 LIMIT 5;
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy
0000003	Field1	/J0t0jg1YEBJDQHbCixW
0000003	Field2	OPvQm03zjv9Lq2fgPkqn
0000003	Field3	Y3hLyea2gMgMpLsM4B0L
0000004	Field1	qUCWcbYquc3O/KXUJ0m3
0000004	Field2	inu5Qkfo/RPEKQNs0vDz
0000004	Field3	paKjwIVOpZWI3dNOkoH4
0000005	Field1	OZzPWB6dULQ/9xDNxOZp
0000005	Field2	gPkqnRHGP3AgS0P2I7Yt
0000005	Field3	VNPLdZmad8qVY3hLyea2
0000006	Field1	AQpbgPdxPPEE0fhCCHXF
0000006	Field2	lta9Jrd1qSo1ypkCzfbN
0000006	Field3	bYvGNdqz307A4pHykJ69
SELECT * FROM LoadTest OFFSET 1 LIMIT 5 RETURN_DELETES;
0000002	Field1	Xj4aa7yqpV5zfGV/5xw8
0000002	Field2	LVaPV+c/kNLJnK/O6pR4
0000002	Field3	hJjg4yZEguV5FHy8hToy
0000003	Field1	/J0t0jg1YEBJDQHbCixW
0000003	Field2	OPvQm03zjv9Lq2fgPkqn
0000003	Field3	Y3hLyea2gMgMpLsM4B0L
0000004	Field1	qUCWcbYquc3O/KXUJ0m3
0000004	Field2	inu5Qkfo/RPEKQNs0vDz
0000004	Field3	paKjwIVOpZWI3dNOkoH4
0000005	Field1	OZzPWB6dULQ/9xDNxOZp
0000005	Field2	gPkqnRHGP3AgS0P2I7Yt
0000005	Field3	VNPLdZmad8qVY3hLyea2
0000006	Field1	AQpbgPdxPPEE0fhCCHXF
0000006	Field2	lta9Jrd1qSo1ypkCzfbN
0000006	Field3	bYvGNdqz307A4pHykJ69
